Daytime-Operations/Auxiliary-Telescope/AuxTel-E-Stop-Procedure/E-Stop-Procecure.rst

```diff
@@ -0,0 +1,118 @@
+.. |author| replace:: *David Sanmartim*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. _E-Stop-Procedure:
+
+####################
+E-Stop Procedure
+####################
+
+
+.. _E-Stop-Procedure-Overview:
+
+Overview
+========
+
+The emergency stop procedure plays a crucial role in ensuring the safety of Rubin Observatory. 
+It is typically used in emergency situations or as a safety measure. 
+It works by cutting off the electrical power to the various components of the telescope, 
+including the mount and dome, which stops them from being moved either by software commands or manually.
+
+The emergency stop can be triggered by users in certain circumstances, but it can also be activated automatically in the event of strong earthquakes 
+(currently not implemented), as well as in response to safety interlocks and software-based emergencies. 
+While the primary objective of the emergency e-stop procedure is to provide a quick and reliable means of halting the movement of the telescope and dome to avoid accidents and injuries, 
+it also serves the purpose of safeguarding the system.
+
+Below you will find a brief overview of the emergency stop procedure, covering:
+
+- :ref:`The location of the relevant switches, <E-Stop-Procedure-Where-to-Find-E-stop-Switches>`
+- :ref:`How to engage/release the emergency stop, and <E-Stop-Procedure-How-Engage-Disengage-E-stop-System>`
+- :ref:`How to determine its current status <E-Stop-Procedure-Verifying-Status-E-stop-System>`.
+  
+
+.. _E-Stop-Procedure-Where-to-Find-E-stop-Switches:
+
+Where to Find the E-stop Switches
```

Comment:
A question "E-stop" should be capitalized?

Reply:
That is a good question. Looking closer into [tstn-004](https://tstn-004.lsst.io/#e-stop-reset-procedure) I see it appears capitalized (except for one occurrence). What do you think? Do you agree with full capitalization?